test/support/pages/HeaderPage.js

```diff
@@ -0,0 +1,80 @@
+// in test/support/pages/HeaderPage.js
+define(function (require) {
+
+  var Common = require('./Common');
+  var url = require('intern/dojo/node!url');
+  var expect = require('intern/dojo/node!expect.js');
+
+  var common;
+
+  // the page object is created as a constructor
+  // so we can provide the remote Command object
+  // at runtime
+  function HeaderPage(remote) {
+    this.remote = remote;
+    common = new Common(this.remote);
+
+  }
+
+  var defaultTimeout = 5000;
+
+  HeaderPage.prototype = {
+    constructor: HeaderPage,
+
+    clickDiscover: function clickDiscover() {
+      common.log('Click Discover tab');
+      return this.remote
+        .setFindTimeout(defaultTimeout)
+        .findByCssSelector('a[href*=\'discover\']')
+        .then(function (tab) {
+          return tab.click();
+        });
+    },
+
+    clickVisualize: function clickVisualize() {
+      var self = this.remote;
+      common.log('Click Visualize tab');
+      return common.tryForTime(5000, function () {
+        return self
+          .setFindTimeout(defaultTimeout)
+          .findByCssSelector('a[href*=\'visualize\']')
+          .then(function (tab) {
+            return tab.click()
+              .then(function checkUrl() {
+                return self
+                  .getCurrentUrl()
+                  .then(function logUrl(currentUrl) {
+                    common.log('currentUrl = ' + currentUrl);
+                    expect(currentUrl).to.contain('visualize');
```

Comment:
I removed it.  It was really some troubleshooting steps when I was having a problem running multiple visualize tests one after another.  But I don't think I needed it and those tests aren't in this PR so I removed it.